app/src/main/java/org/mifos/mobile/ui/transfer_process/TransferProcessScreen.kt

```diff
@@ -0,0 +1,261 @@
+            .fillMaxWidth()
+            .padding(16.dp),
+        colors = CardDefaults.cardColors(
+            containerColor = MaterialTheme.colorScheme.background
+        ),
+        border = BorderStroke(1.dp, Color.LightGray)
+    ) {
+        Column(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(14.dp)
+        ) {
+            Column(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(bottom = 12.dp),
+                horizontalAlignment = Alignment.CenterHorizontally
+            ) {
+                Text(text = stringResource(id = R.string.amount),
+                    color = MaterialTheme.colorScheme.primary)
+
+                Text(text = payload?.transferAmount.toString())
+            }
+            Text(text = stringResource(id = R.string.transfer_from_savings),
+                fontWeight = FontWeight(500),
+                color = Color.Gray
+            )
+
+            Text(text = stringResource(id = R.string.pay_to),
+                modifier = Modifier.padding(top = 8.dp),
+                color = MaterialTheme.colorScheme.primary
+            )
+            Text(text = payload?.fromAccountNumber.toString(),
+                modifier = Modifier
+                    .padding(top = 4.dp, bottom = 2.dp)
+            )
+            Divider()
+
+            Text(text = stringResource(id = R.string.pay_from),
+                modifier = Modifier.padding(top = 8.dp),
+                color = MaterialTheme.colorScheme.primary
+            )
+            Text(text = payload?.fromAccountNumber.toString(),
+                modifier = Modifier
+                    .padding(top = 4.dp, bottom = 2.dp)
+            )
+            Divider()
+
+            Text(text = stringResource(id = R.string.date),
+                modifier = Modifier.padding(top = 8.dp),
+                color = MaterialTheme.colorScheme.primary
+            )
+            Text(text = payload?.transferDate.toString(),
+                modifier = Modifier
+                    .padding(top = 4.dp, bottom = 2.dp)
+            )
+            Divider()
+
+            Text(text = stringResource(id = R.string.remark),
+                modifier = Modifier.padding(top = 8.dp),
+                color = MaterialTheme.colorScheme.primary
+            )
+            Text(text = payload?.transferDescription.toString(),
+                modifier = Modifier
+                    .padding(top = 4.dp, bottom = 2.dp)
+            )
+            Divider()
+
+            Spacer(modifier = Modifier.height(24.dp))
+
+            Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.End
+            ) {
+                Row(
+                    horizontalArrangement = Arrangement.spacedBy(30.dp),
+                ) {
+                    Button(
+                        onClick = {
+                                  cancel()
```

Comment:
![image](https://github.com/openMF/mifos-mobile/assets/78101731/3e65d4ad-45f0-4c8b-a606-c7b6e618d761)
you didn't implement this snackbar.. Original (XML) one has it... add this as well